src/nvim/main.c

```diff
@@ -1907,7 +1922,7 @@ static void main_start_gui(void)
 int
 process_env (
     char_u *env,
-    int is_viminit             /* when TRUE, called for VIMINIT */
+    int is_nviminit             /* when TRUE, called for NVIMINIT */
```

Comment:
Internal identifiers should continue to use `vim` as a generic "protocol" name. `is_viminit` is fine.